main.tf

```diff
@@ -0,0 +1,75 @@
+resource "azurerm_resource_group" "this" {
+  name     = var.resource_group.name
+  location = var.location
+  tags = merge(
+    try(var.tags),
+    tomap({
+      "Resource Type" = "Resource Group"
+    })
+  )
+}
+
+resource "azurerm_log_analytics_workspace" "this" {
+  name                            = var.log_analytics_workspace.name
+  resource_group_name             = azurerm_resource_group.this.name
+  location                        = var.location
+  allow_resource_only_permissions = var.log_analytics_workspace.allow_resource_only_permissions
+  sku                             = var.log_analytics_workspace.sku
+  tags = merge(
+    try(var.tags),
+    tomap({
+      "Resource Type" = "Log Analytics Workspace"
+    })
+  )
+}
+
+module "storage_account" {
+  source = "github.com/schubergphilis/terraform-azure-mcaf-storage-account.git" #TODO: change source
```

Comment:
should this still be todo or ?

Reply:
https://github.com/schubergphilis/terraform-azure-mcaf-monitoring/issues/3 added as an issue for now